presto-docs/src/main/sphinx/functions/tdigest.rst

```diff
@@ -88,3 +88,13 @@ Functions
     its component parts. These include arrays of the centroid means and weights,
     the compression factor, and the maximum, minimum, sum and count of the
     values in the digest.
+
+.. function:: construct_tdigest(centroid_means array<double>, centroid_weights array<double>, compression double, min double, max double, sum double, count bigint) -> tdigest<double>
+
+    Returns the ``tdigest`` which is composed of all of the internal parameters
+    that are used to generate a ``tdigest`` object: arrays of the centroid means
+    and weights, the compression factor, and the maximum, minimum, sum and count of the
+    values in the digest. This is an inverse of ``destructure_tdigest``.
+
+    Particularly useful if a tdigest is computed externally to Presto but still needs
```

Comment:
Nit: Fragment.  Maybe "This function is particularly useful for adding externally-created tdigests to Presto."?